firestore/angular-rewrite/.vscode/launch.json

```diff
@@ -1,3 +1,19 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 {
```

Comment:
Will this automatically run `npm start` ?

Reply:
I was expecting something that looks like the config in the link below - though if this works then there's no need to overcomplicate it

https://stackoverflow.com/questions/34835082/how-to-debug-using-npm-run-scripts-from-vscode

Reply:
> Will this automatically run `npm start` ?

Yes; This script automatically runs the `npm start` command when the green debug play button is pressed. There is also a dropdown that enables the user to run `npm run production` when the `Launch Quickstart  [Production]` is selected, but `Launch Quickstart [Emulators]` (which runs `npm start`) is the default.

Reply:
LGTM